<ui version="4.0" >
 <class>PluginExecutorDialog</class>
 <widget class="QDialog" name="PluginExecutorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Plugin executor</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>250</y>
     <width>471</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title" >
    <string>Dependence</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>190</y>
      <width>261</width>
      <height>59</height>
     </rect>
    </property>
    <property name="title" >
     <string>Execution process</string>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="includeButton" >
       <property name="text" >
        <string>&amp;Include</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="excludeButton" >
       <property name="text" >
        <string>E&amp;xclude</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>25</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="executeButton" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>&amp;Execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeWidget" name="executionTreeWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="indentation" >
     <number>20</number>
    </property>
    <property name="rootIsDecorated" >
     <bool>true</bool>
    </property>
    <column>
     <property name="text" >
      <string>Depends on</string>
     </property>
     <property name="icon" >
      <iconset>../../dd/doghat1.jpg</iconset>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Type</string>
     </property>
    </column>
    <item>
     <property name="text" >
      <string>SimpleCalculation</string>
     </property>
     <property name="text" >
      <string>calculation</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>EasyJavaCalc</string>
     </property>
     <property name="text" >
      <string>calculation</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>CalculationUnion</string>
     </property>
     <property name="text" >
      <string>calculation</string>
     </property>
     <item>
      <property name="text" >
       <string>SimpleCalculation</string>
      </property>
      <property name="text" >
       <string>calculation</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>EasyJavaCalc</string>
      </property>
      <property name="text" >
       <string>calculation</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QGroupBox" name="depParamsGroupBox" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>20</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy>
      <hsizetype>5</hsizetype>
      <vsizetype>5</vsizetype>
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>Dependent plugin parameters</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="paramsGroupBox" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>471</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title" >
    <string>Plugin parameters</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
